### DISCUSSION TITLE
Open issues (Great, but)

### DISCUSSION THREAD

**Discussion by snuup** [dotnet/interactive#3768] [General]
Long time dotnet programmer (since version 0.9), C# and F# here. Also work in data visualisation research and face data science tasks besides coding every other day. I love the idea of notebooks but every time  try them they are not satisfying.

Installed Polyglot Notebooks v1.0.5565010 in VSCode.

**Syntax Highlighting not working** for any languange (not in C#, F#, JS, HTML)
I would have to get used to **Ressource Management** here: close connections or files at end of each cell. But: cannot use "use" in F# cells, hmmm.
**Intellisense** works not always and not as direct as compared to using f# interactive (fsx files).

So as for now I stick to F# interactive and visualisations in browser where I send data to (own implementation) or Plotly.Net to visualize in Browser. That said, I sometimes face challanges with heterogenous data in F# and would prefer a dynamic language like JS or Python. And here the Polyglot idea would be so great.

Would love to see the final poolishing here ad use notebooks.

**Comment by jonsequitur** [comment]
Are you using a dev container? (Syntax highlighting has a known bug there : #2683.) What OS and VS Code version are you on?

F# completions and syntax highlighting are generally working so there might be something specific going on with your setup.

![image](https://github.com/user-attachments/assets/cd937d72-6016-4feb-8ca5-e1ea5cd032aa)
